src/test/java/recipeditor/command/ViewCommandTest.java

```diff
@@ -1,10 +1,75 @@
 package recipeditor.command;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import recipeditor.Recipeditor;
+import recipeditor.exception.RecipeNotFoundException;
+import recipeditor.recipe.Ingredient;
+import recipeditor.recipe.Recipe;
+import recipeditor.recipe.RecipeList;
+
+import javax.swing.text.View;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class ViewCommandTest {
 
+    @BeforeAll
+    public static void setUp() {
+        Recipe recipe = new Recipe("test_title", "test_description");
+        recipe.addIngredient(new Ingredient("ingredient", 100.0, "g"));
+        recipe.addStep("step 1");
+        recipe.addStep("finshing step ~");
+        RecipeList.addRecipe(recipe);
+    }
+
+    @AfterAll
+    public static void tearDown() {
```

Comment:
Maybe you can change the function name to a more relevant one, such as exit.

Reply:
tearDown() stands for dissemble the testing environment, so other code will not be affected by the testing environment.

Reply:
My apologies I didn't know that this was the standard naming in testing!